components/list/list-item-checkbox-mixin.js

```diff
@@ -45,8 +50,11 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(s
 				display: block;
 				height: 100%;
 			}
-			:host([selection-disabled]) .d2l-checkbox-action {
-				cursor: default;
+			:host([selection-disabled]) [slot="control-action"] {
+				pointer-events: none;
```

Comment:
A potential issue is that if there is a case that's using interactive content and also selection and selection-disabled then the interactive content will no longer work.

I don't know that there are cases of that at this point, but something to watch out for.